packages/react-components/src/components/ChatMessage/ChatMessageContent.tsx

```diff
@@ -294,3 +296,22 @@ const processHtmlToReact = (props: ChatMessageContentProps): JSX.Element => {
   };
   return <>{parse(props.message.content ?? '', options)}</>;
 };
+
+const decodeEntities = (encodedString: string): string => {
+  const translate_re = /&(nbsp|amp|quot|lt|gt);/g;
+  const translate: Record<string, string> = {
+    nbsp: ' ',
+    amp: '&',
+    quot: '"',
+    lt: '<',
+    gt: '>'
+  };
+  return encodedString
+    .replace(translate_re, function (match, entity) {
+      return translate[entity];
+    })
+    .replace(/&#(\d+);/gi, function (match, numStr) {
```

Comment:
Could you add some comments/examples here?

Reply:
thanks added